src/js/pie_exploded.js

```diff
@@ -219,31 +219,20 @@ export class PieExploded {
 		}
 
 		function relaxLabels() {
-			let again,
-				thisLabel,
-				thisLabelDoc,
-				y1,
-				thatLabel,
-				thatLabelDoc,
-				y2,
-				deltaY,
-				sign,
-				adjustment,
-				labelForLine;
+			let again, thisLabelDoc, y1, thatLabelDoc, y2, deltaY, sign, adjustment, labelForLine;
```

Comment:
@jdhoffa maybe you could run prettier on all JS files in a monolithic make-things-pretty PR?

Automatic processes making unrelated changes to PRs makes things more difficult to review/understand/track/etc.

This is pretty minor, so I'll let that pass, but if it wants to do major reformatting on a file I touched, I'll be jumping through hoops to avoid that.

Reply:
This wasn't automatic, I did it manually, but happy not to!
Basically the linter will fail on every PR you open unless you style it yourself, so there's two options. 

* You can install `prettier` (`npm install -g prettier`) and use that to style your PRs
* I can go back and manually style your PRs for you

Reply:
But I don't want to merge PRs that have CI failures... especially given all the work I put in to get us to a passing linter

Reply:
> But I don't want to merge PRs that have CI failures... especially given all the work I put in to get us to a passing linter

which means that whether or not you run it manually or it runs automatically, ultimately we are forced to include completely unrelated formatting changes that it suggests to our PRs

Reply:
The formatting is entirely related to this PR... the linter caught something you missed. 

You removed the `thisLabel` and `thatLabel` arguments, but you didn't remove their corresponding `let` statements above. 

If I hadn't run it, that tech debt would have been merged into `main`

Reply:
## Before
``` js
let again,
thisLabel,     <- this gets removed
thisLabelDoc,
y1,
thatLabel,     <- that gets removed
thatLabelDoc,
y2,
deltaY,
sign,
adjustment,
labelForLine;
```

## After
Notice how `thisLabel` and `thatLabel` are no longer initialized:
``` js
let again, thisLabelDoc, y1, thatLabelDoc, y2, deltaY, sign, adjustment, labelForLine;
```

Reply:
If I tried to do a "monolith" PR to address this, I would have all of a sudden had to much more carefully consider if removing `thisLabel` and `thatLabel` was a reasonable thing to do... 

Dealing with it in the PR means we have the full context and understand that it is reasonable.

Reply:
yes, the **linter** caught something that I missed, the earlier var declaration, and I followed with a commit removing them https://github.com/RMI-PACTA/pacta-dashboard-svelte/pull/115/commits/8c996d4e674aee8675be293dd90259ea88550d82

then the linter passed, but specifically `prettier` had a problem

we started with
``` javascript
let again,
				thisLabel,
                                thisLabelDoc,
				y1,
                                thatLabel,
				thatLabelDoc,
				y2,
				deltaY,
				sign,
				adjustment,
				labelForLine;
```

I changed it to this

``` javascript
let again,
				thisLabelDoc,
				y1,
				thatLabelDoc,
				y2,
				deltaY,
				sign,
				adjustment,
				labelForLine;
```

then prettier did this

``` javascript
let again, thisLabelDoc, y1, thatLabelDoc, y2, deltaY, sign, adjustment, labelForLine;
```

Reply:
I think you're mixing up what the linter does and what prettier does, or combining them as one even though they are actually different.